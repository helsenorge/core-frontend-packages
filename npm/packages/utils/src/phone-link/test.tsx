import * as React from "react";
import { shallow } from "enzyme";
import PhoneLink from ".";

describe("PhoneLink", () => {
  it("PhoneLink renders without crashing", () => {
    const wrapper = shallow(<PhoneLink />);
    expect(wrapper.render()).toMatchSnapshot();
  });
});
