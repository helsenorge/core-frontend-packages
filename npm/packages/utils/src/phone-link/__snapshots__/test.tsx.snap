// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhoneLink PhoneLink renders without crashing 1`] = `null`;
